packages/ai-bot/main.ts

```diff
@@ -231,6 +250,17 @@ Common issues are:
           return;
         }
 
+        let eventLock = await acquireLock(
```

Comment:
In case there are many bot instances only the first one that succeeds writing a row to this table will acquire the lock for this event. The operation is atomic and with a constraint so there is no chance of multiple bots getting the lock for the same event